.github/workflows/pr-tests.yml

```diff
@@ -11,7 +11,7 @@ on:
 env:
   CARGO_TERM_COLOR: always
   POWDR_GENERATE_PROOFS: "true"
-  MAX_DEGREE_LOG: "10"
+  MAX_DEGREE_LOG: "20"
```

Comment:
Had to increase, because otherwise the RISC-V tests would run only for 256 rows... Now, it is $2^{18}$, as before this PR. It doesn't seem like it has a big impact on test time (we don't have too many dynamic VADCOP tests and the RISC-V tests are slow anyway).

Reply:
we should also add log 22 tests at some point

Reply:
After #1667, we can configure the range for each test and machine individually, should be much better than this global setting.